### ISSUE TITLE
YAIE #2

### ISSUE THREAD

**Issue by solson** [nixeagle/ooc-mode#18]
Expected indentation:

```
        match scancode {
            // Shift key press
            case 0x2A =>
                shift = true

            // Shift key release
            case 0xAA =>
                shift = false

            // Any other scan code
            case =>
                chr := (shift ? uppercase : lowercase)[scancode] as Char
                Bochs debug("Char: %c" format(chr))
                chr print()
        }
```

ooc-mode indentation:

```
        match scancode {
            // Shift key press
            case 0x2A =>
                shift = true

            // Shift key release
            case 0xAA =>
                shift = false

            // Any other scan code
            case =>
            chr := (shift ? uppercase : lowercase)[scancode] as Char
            Bochs debug("Char: %c" format(chr))
            chr print()
        }
```

Note that it doesn't indent for the final "default" case.

**Comment by nixeagle** [comment]
This seems to work for me

```
match scancode {
    // Shift key press
    case 0x2A =>
        shift = true

        // Shift key release
    case 0xAA =>
        shift = false

        // Any other scan code
    case =>
        chr := (shift ? uppercase : lowercase)[scancode] as Char
        Bochs debug("Char: %c" format(chr))
        chr print()
}
```

Are you doing something different with where the pointer is at? I really can't seem to replicate the described effect. However I've uncovered a bug where if I have the pointer at the end of `chr := (shift ? uppercase : lowercase)[scancode] as Char` and hit tab it will indent too far, but this does not look like what you are describing.

**Comment by nixeagle** [comment]
Indent extra lines after case statements.

Closed by 04ae19c1192aa6dfa9847a261ffea7c8dd0ce772
